README.md

```diff
@@ -9,11 +9,10 @@
 [![codecov](https://codecov.io/gh/PyGithub/PyGithub/branch/master/graph/badge.svg)](https://codecov.io/gh/PyGithub/PyGithub)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-PyGitHub is a Python library to access the [GitHub API v3] and [Github Enterprise API v3].
+PyGitHub is a Python library to access the [GitHub REST API].
 This library enables you to manage [GitHub] resources such as repositories, user profiles, and organizations in your Python applications.
 
-[GitHub API v3]: https://developer.github.com/v3
-[Github Enterprise API v3]: https://developer.github.com/enterprise/v3/
+[GitHub REST API]: https://docs.github.com/en/free-pro-team@latest/rest
```

Comment:
I think `https://docs.github.com/rest` is more intuitive, it's just a 301
```suggestion
[GitHub REST API]: https://docs.github.com/rest
```

Reply:
Your suggested URL redirects to https://docs.github.com/en/free-pro-team@latest/rest - it is the same URL that I proposed in the original Pull request.